tests/xcodeproj/build.sh

```diff
@@ -2,6 +2,4 @@ set -eux
 
 cd $(dirname $0)
 
-xcodebuild -project Single-Application-Project.xcodeproj -quiet
-export SIM_DEVICE_ID=`xcodebuild -project Single-Application-Project.xcodeproj -scheme "Single-Application-UnitTests" -showdestinations | grep "platform:iOS Sim" | head -1 | ruby -e "puts STDIN.read.split(',')[1].split(':').last"`
-xcodebuild -project Single-Application-Project.xcodeproj -scheme "Single-Application-UnitTests"  test-without-building -destination "id=$SIM_DEVICE_ID"
```

Comment:
why not run the test?

Reply:
Since we validate that the scheme in the xcodeproject is associated with the correct set of env vars and cli args (by running git diff on the project.pbxproj file), we already ensure that bazel has correctly associated schemes with the right set of env and args.